### PR TITLE
chore(intl-format-cache): add benchmark for intl-format-cache

### PR DESCRIPTION
**Pull request by longlho**

### CODE REVIEW

File: packages/intl-format-cache/README.md [hunk 1]
```diff
@@ -49,6 +49,36 @@ console.log(nf1.format(1000)); // => "1,000"
...
+fast-memoize x 1,166,894 ops/sec ±0.30% (94 runs sampled)
+intl-format-cache x 1,097,319 ops/sec ±0.70% (94 runs sampled)
+not cached x 15,613 ops/sec ±2.03% (82 runs sampled)
+--- DateTimeFormat cache get: Fastest is fast-memoize ---
+
+fast-memoize x 93,746 ops/sec ±0.25% (93 runs sampled)
+intl-format-cache x 95,376 ops/sec ±0.43% (96 runs sampled)
+--- IntlMessageFormat cache set: Fastest is intl-format-cache ---
+
+fast-memoize x 91,986 ops/sec ±2.50% (91 runs sampled)
```
Comment (redonkulus) [inline packages/intl-format-cache/README.md]: this is surprisingly low

Comment (longlho) [reply packages/intl-format-cache/README.md]: I fixed it. Turns out `fast-memoize` for some reason doesn't deal w/ ES6 `class` very well

Comment (longlho) [reply packages/intl-format-cache/README.md]: If we just normalize to regular fn, `fast-memoize` & `intl-format-cache` is basically the same. `Intl.NumberFormat` works w/ & w/o `new` per [spec](https://www.ecma-international.org/ecma-402/2.0/#sec-Intl.NumberFormat).

File: packages/intl-format-cache/tests/benchmark.ts [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  onCycle,
+  onError: console.log,
+  onComplete
+})
+  .add('fast-memoize', () => dtm('de', { month: 'short' }))
+  .add('intl-format-cache', () => dtfc('de', { month: 'short' }))
+  .add('not cached', () => new Intl.DateTimeFormat('de', { month: 'short' }))
+  .run();
+
+function createIntlMessageFormat (...args: ConstructorParameters<typeof IntlMessageFormat>) {
```
Comment (longlho) [inline packages/intl-format-cache/tests/benchmark.ts]: https://github.com/formatjs/formatjs/pull/38 should get rid of this

**Review by redonkulus — APPROVED** [review]

**Pull request merged**